client/src/public/ts/socket.ts

```
@@ -115,6 +117,137 @@ socket.addEventListener("message", (event: any) => {
+
+      // if cache doesn't match, redo html
+
+      // clear cache
+      variables.multiplayerLastGameRankings.playerListCache.playerCount = 0;
+      variables.multiplayerLastGameRankings.playerListCache.registeredPlayers.clear();
+
+      const selector =
+        "#main-content__multiplayer-intermission-screen-container__game-status-ranking";
+      $(selector).empty();
+      const placements = message.data;
+      for (let placement of placements) {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'placement' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)